app/process/maxspeed/maxspeed.lua

```diff
@@ -0,0 +1,434 @@
+  --  dann alle linien die (TODO) vollständig / am meisten / … in der fläche fläche sind
+  --  (tendentizell dafür nicht schneiden, weil wir am liebsten die OSM ways so haben wie in OSM)
+  --  und dann können wir in sql "maxspeed" "maxspeed_source='infereed from landuse'"
+  --  UND dann auch einen "_todo="add 'maxspeed:source=DE:urban' to way"
+  -- hinweis: außerstädtisch extrapolieren wir aber keine daten, da zu wenig "richtig"
+
+
+  -- "is_present": Skip-Values umbauen, so dass alle maxspeed-relevanten daten im haupt datensatz sind
+  --    wenn primärdaten vorhanden, dann is_present=true
+
+  -- Sort the paths depending on whether they have a maxspeed tag in the ToDo table or in the results table
+  SortIntoTables(object)
+  -- Freshness of data
+  CheckDate(object, object.tags.maxspeed)
+
+  -- all tags that are shown on the application
+  local allowed_tags =
+      Set(
+          {
+              "_skip",
+              "_skipNotes",
+              "is_present",
+              "_combined_fresh_age_days",
+              "_combined_is_fresh",
+              "_update_fresh_age_days",
+              "_update_is_fresh",
+              "_centerline",
+              "_maxspeed_source",
+              "_todo",
+              "fresh",
+              "fresh_age_days",
+              "_freshNotes",
+              "bicycle_road",
+              "bicycle",
+              "category",
+              "cycleway",
+              "unclassified",
+              "secondary",
+              "residential",
+              "highway",
+              "maxspeed",
+              "maxspeed:backward",
+              "maxspeed:forward",
+              "maxspeed:conditional", -- show if present; details TBD
+              "source:maxspeed", -- only for debugging in webapp
+              "maxspeed:type", -- only for debugging in webapp
+              "zone:maxspeed", -- only for debugging in webapp
+              "zone_traffic",
+              "traffic_sign",
+              "maxspeed_split",
+              "source_maxspeed_forward",
+              "source_maxspeed_backward",
+              "source_maxspeed" -- only for debugging in webapp
+          }
+      )
+
+  FilterTags(object.tags, allowed_tags)
+  AddMetadata(object)
+  AddUrl("way", object)
+
+  -- insert the ways in theri respective table based on the tags _skip or _todo
+  if object.tags._skip == true then
+    excludeTable:insert(
+        {
+            tags = object.tags,
+            geom = object:as_linestring()
+        }
+    )
+  end
+
+  if object.tags._todo == true and object.tags._skip == false then
+    todoTable:insert(
+        {
+            tags = object.tags,
+            geom = object:as_linestring()
+        }
+    )
+  end
+
+  if object.tags._skip == false and object.tags._todo == false then
+    table:insert(
+        {
+            tags = object.tags,
+            geom = object:as_linestring()
+        }
+    )
+  end
+end
+
+function FindBicycleroads(object)
```

Comment:
This is a bit mixed up with the category logic and a bit hard to understand whats going on (filtering and max speed definition mixed)